src/test/regress/expected/multi_schema_support.out

```diff
@@ -794,7 +794,7 @@ SELECT * FROM pg_dist_shard_placement WHERE shardid = 1190000;
  shardid | shardstate | shardlength | nodename  | nodeport 
 ---------+------------+-------------+-----------+----------
  1190000 |          1 |        8192 | localhost |    57638
- 1190000 |          1 |           0 | localhost |    57637
+ 1190000 |          1 |        8192 | localhost |    57637
```

Comment:
Our old master_copy functions does not correctly sets shardlength and new approach fixes it as side-effect :)